src/main/webapp/app/code-editor/actions/code-editor-resolve-conflict-modal.component.html

```diff
@@ -0,0 +1,21 @@
+<form class="conflict-modal" #deleteForm="ngForm" name="deleteForm" (ngSubmit)="resetRepository()">
+    <div class="conflict-modal__header">
+        <h4 jhiTranslate="artemisApp.editor.conflict.conflictExplanationShort"></h4>
+    </div>
+    <div class="conflict-modal__body">
+        <jhi-alert-error></jhi-alert-error>
+        <div jhiTranslate="artemisApp.editor.conflict.conflictExplanationLong"></div>
+        <span class="conflict-modal__body__warning">
```

Comment:
why does the class contain double underscores like "__"?

Reply:
This is part of the [BEM css methodology](http://getbem.com/introduction/).
The __ structures Blocks (e.g. here are some examples: https://bluegg.co.uk/writing/building-my-health-skills-part-3).